mvnw.cmd

```diff
@@ -1,145 +1,145 @@
-@REM ----------------------------------------------------------------------------

```

Comment:
never used windows for development/builds :D last time it ends with workarounds in java specific to windows bugs...